fancontrol/fancontrol.cpp

```diff
@@ -82,20 +84,18 @@ FANCONTROL::FANCONTROL(HINSTANCE hinstapp)
 	Runs_as_service(FALSE),
 	ActiveMode(false),
 	ManFanSpeed(7),
-	UseTWR(0),
```

Comment:
hm not sure where this ended up

Reply:
Yup, I'm still going over all this with a fine toothed comb myself.